docs/README.md

```diff
@@ -6,25 +6,33 @@
 
 **Welcome to the _ZARP_ documentation pages!**
 
-_ZARP_ is a [Snakemake][snakemake] workflow ...
-
-**TODO:** Complete intro/summary section
-
-**NOTES TO SELF:**
-
-- Include references in `/includes/references.md`
-- Include abbreviations in `includes/abbreviations.md` (example: EAWNKWIMPR)
-- Include emojis (example: :rocket:)
-- Check out the [ZARP-cli docs pages](https://zavolab.github.io/zarp-cli/)
-for inspiration.
-- Check out [MKDocs](https://www.mkdocs.org/) and [Material for
-MkDocs](https://squidfunk.github.io/mkdocs-material/) docs for more info
+_ZARP_ is a generic RNA-Seq analysis workflow that allows users to process and analyze 
+Illumina short-read sequencing libraries with minimum effort. Better yet: With our
+companion [**ZARP-cli**](https://github.com/zavolanlab/zarp-cli) command line
+interface, you can start ZARP runs with the simplest and most intuitive commands.
+
+_RNA-seq analysis doesn't get simpler than that!_
+
+ZARP relies on publicly available bioinformatics tools and currently handles
+single or paired-end stranded bulk RNA-seq data. The workflow is developed in
```

Comment:
When considering the first sentence of the next paragraph, this sentence is completely redundant. Probably best to delete this sentence, and merge the remaining sentence with the following paragraph:

> ZARP is developed in Snakemake, a widely used workflow management system in the bioinformatics community. The workflow will pre-process, [...]

Reply:
Updated. Thank you.